msg/js/es-419.js

```diff
@@ -0,0 +1,271 @@
+Blockly.Msg["LOOKS_SWITCHBACKDROPTO"] = "cambiar fondo a %1";
+Blockly.Msg["LOOKS_GOTOFRONTBACK"] = "ir a la capa %1 ";
+Blockly.Msg["LOOKS_GOTOFRONTBACK_FRONT"] = "del frente";
+Blockly.Msg["LOOKS_GOTOFRONTBACK_BACK"] = "de atrás";
+Blockly.Msg["LOOKS_GOFORWARDBACKWARDLAYERS"] = "ir %1 %2 capas";
+Blockly.Msg["LOOKS_GOFORWARDBACKWARDLAYERS_FORWARD"] = "adelante";
+Blockly.Msg["LOOKS_GOFORWARDBACKWARDLAYERS_BACKWARD"] = "atrás";
+Blockly.Msg["LOOKS_BACKDROPNUMBERNAME"] = "fondo %1";
+Blockly.Msg["LOOKS_COSTUMENUMBERNAME"] = "# de disfraz %1";
+Blockly.Msg["LOOKS_NUMBERNAME_NUMBER"] = "número";
+Blockly.Msg["LOOKS_NUMBERNAME_NAME"] = "nombre";
+Blockly.Msg["LOOKS_SWITCHBACKDROPTOANDWAIT"] = "cambiar fondo a %1 y esperar";
+Blockly.Msg["LOOKS_NEXTBACKDROP"] = "siguiente fondo";
+Blockly.Msg["LOOKS_PREVIOUSBACKDROP"] = "fondo anterior";
+Blockly.Msg["MOTION_MOVESTEPS"] = "mover %1 pasos";
+Blockly.Msg["MOTION_TURNLEFT"] = "girar %1 %2 grados";
+Blockly.Msg["MOTION_TURNRIGHT"] = "girar %1 %2 grados";
+Blockly.Msg["MOTION_POINTINDIRECTION"] = "apuntar en dirección %1";
+Blockly.Msg["MOTION_POINTTOWARDS"] = "apuntar hacia %1";
+Blockly.Msg["MOTION_POINTTOWARDS_POINTER"] = "puntero del ratón";
+Blockly.Msg["MOTION_GOTO"] = "ir a %1";
+Blockly.Msg["MOTION_GOTO_POINTER"] = "puntero del ratón";
+Blockly.Msg["MOTION_GOTO_RANDOM"] = "posición aleatoria";
+Blockly.Msg["MOTION_GOTOXY"] = "ir a x: %1 y: %2";
+Blockly.Msg["MOTION_GLIDESECSTOXY"] = "deslizar en %1 segs a x: %2 y: %3";
+Blockly.Msg["MOTION_GLIDETO"] = "deslizar en %1 segs a x: %2";
+Blockly.Msg["MOTION_GLIDETO_POINTER"] = "puntero del ratón";
+Blockly.Msg["MOTION_GLIDETO_RANDOM"] = "posición aleatoria";
+Blockly.Msg["MOTION_CHANGEXBY"] = "cambiar x en %1";
+Blockly.Msg["MOTION_SETX"] = "fijar x a %1";
+Blockly.Msg["MOTION_CHANGEYBY"] = "cambiar y en %1";
+Blockly.Msg["MOTION_SETY"] = "fijar y a %1";
+Blockly.Msg["MOTION_IFONEDGEBOUNCE"] = "rebotar si toca un borde";
+Blockly.Msg["MOTION_SETROTATIONSTYLE"] = "fijar estilo de rotación %1";
+Blockly.Msg["MOTION_SETROTATIONSTYLE_LEFTRIGHT"] = "izquierda-derecha";
+Blockly.Msg["MOTION_SETROTATIONSTYLE_DONTROTATE"] = "no rotar";
+Blockly.Msg["MOTION_SETROTATIONSTYLE_ALLAROUND"] = "en todas direcciones";
+Blockly.Msg["MOTION_XPOSITION"] = "posición en x";
+Blockly.Msg["MOTION_YPOSITION"] = "posición en y";
+Blockly.Msg["MOTION_DIRECTION"] = "dirección";
+Blockly.Msg["MOTION_SCROLLRIGHT"] = "desplazar a la derecha %1";
+Blockly.Msg["MOTION_SCROLLUP"] = "desplazar arriba %1";
+Blockly.Msg["MOTION_ALIGNSCENE"] = "alinear escena %1";
+Blockly.Msg["MOTION_ALIGNSCENE_BOTTOMLEFT"] = "abajo-izquierda";
+Blockly.Msg["MOTION_ALIGNSCENE_BOTTOMRIGHT"] = "abajo-derecha";
+Blockly.Msg["MOTION_ALIGNSCENE_MIDDLE"] = "medio";
+Blockly.Msg["MOTION_ALIGNSCENE_TOPLEFT"] = "arriba-izquierda";
+Blockly.Msg["MOTION_ALIGNSCENE_TOPRIGHT"] = "arriba-derecha";
+Blockly.Msg["MOTION_XSCROLL"] = "desplazar x";
+Blockly.Msg["MOTION_YSCROLL"] = "desplazar y
```

Comment:
newline in string leading to unterminated parse. In addition to fixing this value, double check your string escaping in script that generates these files.

Reply:
Good catch - I'll add a check as there is no reason for a newline in any of the msg json files.